### ISSUE TITLE
No option to set `Min CheckPoint Count` and `Max CheckPoint Count` on a persistent subscription

### ISSUE THREAD

**Issue by cdavernas** [pyeventsourcing/esdbclient#21]
AFAIK, there's actually no way to configure the `Min CheckPoint Count` and `Max CheckPoint Count` on a persistent subscription, which are critical in many use cases. 

This means that ES is gonna checkpoint acked events only after the default `10` (min/max) events.

**Comment by johnbywater** [comment]
Thanks for raising this issue.

I've added optional arguments `min_checkpoint_count` and `max_checkpoint_count` to methods `create_subscription_to_all()`, `create_subscription_to_stream()`, `update_subscription_to_all()`, and `update_subscription_to_stream()`.

Please try newly released v1.0.19 and see that works for you?

**Comment by cdavernas** [comment]
Oh woaw, super reactive! Thank you so much ❤️ 
Will try tomorrow morning and keep you posted!

**Comment by cdavernas** [comment]
@johnbywater Works like a charm, thank you!

**Comment by johnbywater** [comment]
I had a look at your code. It seems you are writing Asyncio code but using the sync client. There's also an asyncio client in the package, which now has the persistent subscription methods you are using. Maybe try to use that, rather than having your awaitables blocking on requests to the database?

**Comment by cdavernas** [comment]
Oh, i didnt know about the async client! I will use it, thanks @johnbywater!

**Comment by cdavernas** [comment]
@johnbywater I tried using the async client, however I'm facing a problem due to the fact my DI configuration is sync, whereas the construction of the client is async.

I tried several approaches to get the client from within said sync method (`ESEventStore.configure`) but cannot get it working. 

I'm pretty new to Python, my weapon of choice being C#, therefore any help to get it working would be greatly appreciated!

P.S.: To further complexify things, `asyncio.run` won't do the trick, as the configuration method is called from within a running asyncio loop, which can be tested using the OpenBank sample.
